.github/workflows/tests.yml

```diff
@@ -106,12 +86,16 @@ jobs:
           nox --version
           uv --version
 
-      - name: Skip flaky azure, gs remotes if unavailable on macos
+      - name: Skip flaky azure, gs remotes on macOS
         if: runner.os == 'macOS'
-        run: echo 'DATACHAIN_TEST_SKIP_MISSING_REMOTES=azure,gs' >> "$GITHUB_ENV"
+        run: echo 'DISABLE_REMOTES_ARG=--disable-remotes=azure,gs' >> "$GITHUB_ENV"
+
+      - name: Skip all remotes on Windows
+        if: runner.os == 'Windows'
+        run: echo 'DISABLE_REMOTES_ARG=--disable-remotes=all' >> "$GITHUB_ENV"
```

Comment:
Checking if this speeds up Windows test.

Reply:
Looks like it does help.

|  | `main` | PR |
|--------|--------|--------|
| windows-latest/3.9 | 14m 31s | 8m 38s |
| windows-latest/3.12 | 9m 58s |  7m 4s |